.env.example

```diff
@@ -91,10 +91,20 @@ REDIS_SENTINEL_USERNAME=
 REDIS_SENTINEL_PASSWORD=
 REDIS_SENTINEL_SOCKET_TIMEOUT=0.1
 
-DB_USERNAME=postgres
-DB_PASSWORD=difyai123456
-DB_HOST=localhost
-DB_PORT=5432
+DB_TYPE=postgresql
+
+# PostgreSQL configuration
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=difyai123456
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+
+# MySQL configuration
+MYSQL_USER=root
+MYSQL_PASSWORD=difyai123456
+MYSQL_HOST=localhost
+MYSQL_PORT=3306
```

Comment:
Could we use more general terms like USER, PASSWORD, HOST, and PORT here for all database types? And keep POSTGRES_* for backward compatibility.

Reply:
Makes sense to me; let's keep the `MYSQL_` prefix.